src/main/java/org/jitsi/videobridge/Conference.java

```diff
@@ -240,15 +246,21 @@ public Conference(Videobridge videobridge,
                       String id,
                       String focus,
                       String name,
-                      boolean enableLogging)
+                      boolean enableLogging,
+                      String gid)
```

Comment:
The lack of a null-check on `gid`, while the same commit does add such checks for others, implies that `gid`  can be null. Could this be documented explicitly, both in the constructor, as well as in the getter?